.github/workflows/symfony-test.yml

```
@@ -0,0 +1,317 @@
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set IMAGE_NAME for this project
+        run: echo "TEST_IMAGE_NAME=${{ needs.build-and-push-test-image.outputs.matrix_test_image_name }}" >> $GITHUB_ENV
+
+      - name: Copy libs PHP inside project
+        run: cp libs/php ${{ steps.project-paths.outputs.project_path }}/packages
```

actionlint (1.7.4)

property "project-paths" is not defined in object type {}

(expression)